src/chttpd/src/chttpd.erl

```diff
@@ -537,15 +641,22 @@ extract_cookie(#httpd{mochi_req = MochiReq}) ->
 %%% end hack
 
 set_auth_handlers() ->
-    AuthenticationDefault =  "{chttpd_auth, cookie_authentication_handler},
-      {chttpd_auth, default_authentication_handler}",
+    AuthenticationDefault =
+        "{chttpd_auth, cookie_authentication_handler},\n"
+        "      {chttpd_auth, default_authentication_handler}",
     AuthenticationSrcs = couch_httpd:make_fun_spec_strs(
-        config:get("chttpd", "authentication_handlers", AuthenticationDefault)),
+        config:get("chttpd", "authentication_handlers", AuthenticationDefault)
+    ),
     AuthHandlers = lists:map(
-        fun(A) -> {auth_handler_name(A), couch_httpd:make_arity_1_fun(A)} end, AuthenticationSrcs),
-    AuthenticationFuns = AuthHandlers ++ [
-        fun chttpd_auth:party_mode_handler/1 %% must be last
-    ],
+        fun(A) -> {auth_handler_name(A), couch_httpd:make_arity_1_fun(A)} end,
```

Comment:
0

However I would prefer to always have function body on new line
```
fun(A) ->
     {auth_handler_name(A), couch_httpd:make_arity_1_fun(A)}
end,
```

Reply:
+1

Reply:
-1